components/AdminFunc/CreateClass.tsx

```diff
@@ -0,0 +1,189 @@
+import { useState } from 'react'
+import useForm from '../FormElements/useForm'
+import getWeek from '../../lib/getWeek'
+import styled from 'styled-components'
+import gql from 'graphql-tag'
+import { useMutation } from '@apollo/client'
+import { ALL_CLASSES_QUERY } from '../Schedule/Schedule'
+
+const CREATE_SPORTCLASS_MUTATION = gql`
+  mutation CREATE_SPORTCLASS_MUTATION(
+    $name: String!
+    $freeSpots: Int!
+    $status: String!
+    $week: Int!
+    $day: String!
+    $startTime: String!
+    $teacher: String!
+    $duration: String!
+  ) {
+    createSportClass(
+      data: {
+        name: $name
+        freeSpots: $freeSpots
+        status: $status
+        week: $week
+        day: $day
+        startTime: $startTime
+        teacher: $teacher
+        duration: $duration
+      }
+    ) {
+      id
+    }
+  }
+`
+
+export default function CreateClass() {
+  const { inputs, handleChange, clearForm } = useForm({
+    name: '',
+    freeSpots: 10,
+    status: 'AVAILABLE',
```

Comment:
AVAILABLE and NOT AVAILABLE are the equivalent of `TRUE` and `FALSE`. Better to use a boolean value here. Also easier to process